### PR TITLE
[MySql] Restructure test files to speed up daily test pipeline

### PR DESCRIPTION
**Pull request by DaeunYim**
## Description

Submitting a new PR for this with rebased branch. 
This is not customer facing changes and only for daily running pipelines for Orcas team. 

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by msJinLei** [comment]
@DaeunYim 

- Please rebase to the latest generation and  resolve the conflicts
- As you implemented compound test cases where multiple cmdlets are tested.  For test case A, you test both cmdlet a and b, for example. It is better to put A to the  test file of a. In the test file of b, you can skip the implementation, but please add the comments, say the tests are implemented in a's test file.

**Comment by DaeunYim** [comment]
> @DaeunYim
> 
> * Please rebase to the latest generation and  resolve the conflicts
> * As you implemented compound test cases where multiple cmdlets are tested.  For test case A, you test both cmdlet a and b, for example. It is better to put A to the  test file of a. In the test file of b, you can skip the implementation, but please add the comments, say the tests are implemented in a's test file.

I very much agree with this point. Will work on this.

**Comment by msJinLei** [comment]
> > @DaeunYim
> > 
> > * Please rebase to the latest generation and  resolve the conflicts
> > * As you implemented compound test cases where multiple cmdlets are tested.  For test case A, you test both cmdlet a and b, for example. It is better to put A to the  test file of a. In the test file of b, you can skip the implementation, but please add the comments, say the tests are implemented in a's test file.
> 
> I very much agree with this point. Will work on this.

My point is to remove all the test cases file that start with "Az" and put tese cases of them into normal test case files.
But it's OK If you want to keep these files. Please remove related the normal test files. It's no need to keep these files and leave the functions in them with "not implemented".

File: src/MySql/readme.md [hunk 1]
```diff
@@ -62,6 +62,9 @@ directive:
   - from: Microsoft.DBforMySQL/preview/2020-07-01-preview/mysql.json
     where: $
     transform: return $.replace(/\/subscriptions\/\{subscriptionId\}\/resourceGroups\/\{resourceGroupName\}\/providers\/Microsoft\.DBForMySql\/flexibleServers\/\{serverName\}/g, "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DBforMySQL/flexibleServers/{serverName}")
+  - from: swagger-document
```
Comment (msJinLei) [inline src/MySql/readme.md:65]: Remove this line.

**Review by msJinLei — CHANGES_REQUESTED** [review]

**Comment by DaeunYim** [comment]
> Please remove related the normal test files. It's no need to keep these files and leave the functions in them with "not implemented".

I wanted to remove all these files but noticed that these files are re-created when using auto-generation. Would it not re-create when you merge the generation branch into the master branch in the future?

**Comment by msJinLei** [comment]
> > Please remove related the normal test files. It's no need to keep these files and leave the functions in them with "not implemented".
> 
> I wanted to remove all these files but noticed that these files are re-created when using auto-generation. Would it not re-create when you merge the generation branch into the master branch in the future?

yes, they will be re-created when you re-generated the code but wouldn't be created when merge to master.
please address another comments for this PR.

**Comment by DaeunYim** [comment]
> > > Please remove related the normal test files. It's no need to keep these files and leave the functions in them with "not implemented".
> > 
> > 
> > I wanted to remove all these files but noticed that these files are re-created when using auto-generation. Would it not re-create when you merge the generation branch into the master branch in the future?
> 
> yes, they will be re-created when you re-generated the code but wouldn't be created when merge to master.
> please address another comments for this PR.

Lei, I've been missing this comment. Really sorry for late response. Made the change you requested. Sorry about misunderstanding it!

**Review by msJinLei — APPROVED** [review]

**Pull request merged**